client/components/user/UserPrivacy.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <v-layout align-center>
+    <v-card class="card-container">
+      <h1 class="title">Change password</h1>
+      <form @submit.prevent="submit">
+        <v-text-field
+          v-validate="'required|alphanumerical|min:6'"
```

Comment:
Use object form of `validate` exclusively. Apply that to following two fields :bowtie: